osmaxx-py/excerptconverter/tests/__init__.py

```diff
@@ -0,0 +1 @@
+# scripts with names of form 'test*.py' are autodiscovered
```

Comment:
Why is this comment here? I don't think we should document django documentation/python test framework inside here.